cotainr/tests/pack/test_conda_install.py

```diff
@@ -404,15 +404,17 @@ def test_miniforge_still_showing_license(
         # shown to the user when extracting and showing the Miniforge license
         # terms from the installer - just to have an idea that the license is
         # still being shown correctly to the user
-        assert (
-            "Miniforge installer code uses BSD-3-Clause license as stated below."
-        ) in stdout
-        assert (
-            "Miniforge installer comes with a boostrapping executable that is used\n"
-            "when installing miniforge and is deleted after miniforge is installed."
-        ) in stdout
-        assert "conda-forge" in stdout
-        assert "All rights reserved." in stdout
+        miniforgeList = [
+            "Miniforge installer comes",
+            "installing miniforge and is deleted",
+            "bootstrapping executable uses",
+            "micromamba",
+            "conda-forge",
+            "All rights reserved."]
+        for licensePart in miniforgeList:
```

Comment:
This doesn't appear to be formatted using [black](https://black.readthedocs.io/en/stable/) as required by the [cotainr style guide](https://cotainr.readthedocs.io/en/stable/development/style_guide.html). Please run black on the entire change set.

Reply:
I decided to move the answering to a function that returns a boolean in util instead to make it more standardized.
And blackened everything.